src/views/BusinessCase/index.tsx

```diff
@@ -13,34 +13,60 @@ import GeneralProjectInfo from './GeneralProjectInfo';
 import ProjectDescription from './ProjectDescription';
 import AsIsSolution from './AsIsSolution';
 import PreferredSolution from './PreferredSolution';
+import AlternativeSolution from './AlternativeSolution';
 import './index.scss';
 
+// Default data for estimated lifecycle costs
+const defaultEstimatedLifecycle = {
+  year1: [{ phase: '', cost: '' }],
+  year2: [{ phase: '', cost: '' }],
+  year3: [{ phase: '', cost: '' }],
+  year4: [{ phase: '', cost: '' }],
+  year5: [{ phase: '', cost: '' }]
+};
+
+// Default data for a proposed solution
+export const defaultProposedSolution = {
```

Comment:
This is the first time we've ever need to export these "default values". They should probably get moved out of the view at some point, but it's not 100% clear where it belongs. I've heard some projects merging them in the same file as constants or the typescript types. Not really sure, but it's something to think about for future clean up.

If we do end up moving this out, we should also move default values out of the system intake view as well.

Reply:
hmmm, it sounds like we've already got two use cases for a `defaultFormData` file?

Reply:
Yeah I was thinking something like that too. We'd likely have to double back and do this for the System Intake too.

We should take a look at the file structure and/or do some research to see where it makes more sense and take care of this "tech debt" in an upcoming sprint.